Gruntfile.coffee

```diff
@@ -256,9 +256,9 @@ module.exports = (grunt) ->
           level: 9
         expand: true
         cwd: 'dist/'
-        src: ['**/*.{html,xml,css,js,map,svg,otf,ttf,ftl}']
+        src: ['**/{*.{html,xml,css,svg,otf,ttf,ftl},!(settings).js,!(require.js).map}']
```

Comment:
Does this exclude the `.map` files from being included in the tarball?

Reply:
nevermind, I see the following files in `dist/scripts/`:

```
main.js
main.js.gz
main.js.map
main.js.map.gz
require.js
require.js.gz
require.js.map
settings.js
```

(and the `!(settings).js` apparently means "all `.js` files that are not named `settings.js`)